### DISCUSSION TITLE
base URL

### DISCUSSION THREAD

**Discussion by akil-rails** [vuejs/vitepress#1305] [Q&A] [answered]
The documentation talks about setting the [base](https://vitepress.vuejs.org/config/app-configs#base). 

This works for a path such as /credit-cards/ and the site shows up correctly when I run the following: 
``` 
pnpm docs:serve
```

But for a path such as /docs/credit-cards/, I get 404 errors when I run the following:
```
pnpm docs:serve
```

<img width="828" alt="CleanShot 2022-09-08 at 13 00 57@2x" src="https://user-images.githubusercontent.com/469760/189062201-c57f1d08-e66a-441a-b881-a5228c94811f.png">

The goal is to have different repositories for documentation for different products all deployed under the same domain and a common base - such as the following:

```
apibanking.com/docs/payments/  --- one site
apibanking.com/docs/deposits/  --- another site
```

Is this possible to do?

**Comment by brc-dd** [comment, marked as answer]
This appears to be a bug with `polka`/`sirv`. For now consider doing this:

```
  base: '/foo/bar/',
  outDir: '.vitepress/dist/foo/bar',
```

Then instead of `docs:serve`, run:

```
npx serve docs/.vitepress/dist
```
